BlockchainSdkTests/VectorTests/PublicKeyDerivationAddressTests.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  PublicKeyDerivationAddressTests.swift
+//  BlockchainSdkTests
+//
+//  Created by skibinalexander on 25.04.2023.
+//  Copyright © 2023 Tangem AG. All rights reserved.
+//
+
+import XCTest
+import CryptoKit
+import TangemSdk
+import WalletCore
+
+@testable import BlockchainSdk
+
+class PublicKeyDerivationAddressTests: XCTestCase {
+    let addressesUtility = AddressServiceManagerUtility()
+    let testVectorsUtility = TestVectorsUtility()
+}
+
+/*
+ - 0. Compare derivation from vector file with BlockchainSdk.derivationPath(.new)
+ - 1. Obtain MASTER Trustwallet Keys and keys from TrangemSdk and compare keys
+ - 2. Obtain PUBLIC Trustwallet Keys and keys from TrangemSdk and compare keys
+ - 2. Obtain ADDRESSES TeustWallet service and BlockchainSdk service for derivation public keys
```

Comment:
```suggestion
 - 3. Obtain ADDRESSES TrustWallet service and BlockchainSdk service for derivation public keys
```

а в коде получаем только публичный ключ из траста

Reply:
Поправил описание